{"nodes":[{"pos":[986,1084],"content":"The exception that is thrown when an unexpected operation occurs during a cryptographic operation.","needQuote":true,"needEscape":true,"nodes":[{"content":"The exception that is thrown when an unexpected operation occurs during a cryptographic operation.","pos":[0,98]}]},{"pos":[1097,1343],"content":"CryptographicUnexpectedOperationException uses the HRESULT CORSEC_E_CRYPTO_UNEX_OPER, which has the value 0x80131431.       For a list of initial property values for an instance of CryptographicUnexpectedOperationException, see the constructor.","needQuote":false,"needEscape":true,"extradata":"MT","nodes":[{"content":"CryptographicUnexpectedOperationException uses the HRESULT CORSEC_E_CRYPTO_UNEX_OPER, which has the value 0x80131431.       For a list of initial property values for an instance of CryptographicUnexpectedOperationException, see the constructor.","pos":[0,244],"nodes":[{"content":"CryptographicUnexpectedOperationException uses the HRESULT CORSEC_E_CRYPTO_UNEX_OPER, which has the value 0x80131431.","pos":[0,117]},{"content":"For a list of initial property values for an instance of CryptographicUnexpectedOperationException, see the constructor.","pos":[124,244]}]}]},{"pos":[2742,2898],"content":"Initializes a new instance of the <bpt id=\"p1\">&lt;xref href=\"System.Security.Cryptography.CryptographicUnexpectedOperationException\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> class with default properties.","needQuote":true,"needEscape":true,"source":"Initializes a new instance of the <xref href=\"System.Security.Cryptography.CryptographicUnexpectedOperationException\"></xref> class with default properties."},{"pos":[2911,3249],"content":"The following table shows the initial property values for an instance of <xref:System.Security.Cryptography.CryptographicUnexpectedOperationException>.      |Property|Value|   |--------------|-----------|   |<xref:System.Exception.InnerException%2A>|`null`.|   |<xref:System.Exception.Message%2A>|The localized error message string.|","needQuote":false,"needEscape":true,"extradata":"MT","nodes":[{"content":"The following table shows the initial property values for an instance of &lt;xref:System.Security.Cryptography.CryptographicUnexpectedOperationException&gt;.","pos":[0,151],"source":"The following table shows the initial property values for an instance of <xref:System.Security.Cryptography.CryptographicUnexpectedOperationException>."},{"content":"|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|<ph id=\"ph1\">`null`</ph>.|   |&lt;xref:System.Exception.Message%2A&gt;|The localized error message string.|","pos":[157,333],"source":"      |Property|Value|   |--------------|-----------|   |<xref:System.Exception.InnerException%2A>|`null`.|   |<xref:System.Exception.Message%2A>|The localized error message string.|"}]},{"pos":[4723,4886],"content":"Initializes a new instance of the <bpt id=\"p1\">&lt;xref href=\"System.Security.Cryptography.CryptographicUnexpectedOperationException\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> class with a specified error message.","needQuote":true,"needEscape":true,"source":"Initializes a new instance of the <xref href=\"System.Security.Cryptography.CryptographicUnexpectedOperationException\"></xref> class with a specified error message."},{"pos":[4899,5227],"content":"The following table shows the initial property values for an instance of <xref:System.Security.Cryptography.CryptographicUnexpectedOperationException>.      |Property|Value|   |--------------|-----------|   |<xref:System.Exception.InnerException%2A>|`null`.|   |<xref:System.Exception.Message%2A>|The error message string.|","needQuote":false,"needEscape":true,"extradata":"MT","nodes":[{"content":"The following table shows the initial property values for an instance of &lt;xref:System.Security.Cryptography.CryptographicUnexpectedOperationException&gt;.","pos":[0,151],"source":"The following table shows the initial property values for an instance of <xref:System.Security.Cryptography.CryptographicUnexpectedOperationException>."},{"content":"|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|<ph id=\"ph1\">`null`</ph>.|   |&lt;xref:System.Exception.Message%2A&gt;|The error message string.|","pos":[157,323],"source":"      |Property|Value|   |--------------|-----------|   |<xref:System.Exception.InnerException%2A>|`null`.|   |<xref:System.Exception.Message%2A>|The error message string.|"}]},{"pos":[6027,6088],"content":"The error message that explains the reason for the exception.","needQuote":true,"needEscape":true,"nodes":[{"content":"The error message that explains the reason for the exception.","pos":[0,61]}]},{"pos":[7089,7242],"content":"Initializes a new instance of the <bpt id=\"p1\">&lt;xref href=\"System.Security.Cryptography.CryptographicUnexpectedOperationException\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> class with serialized data.","needQuote":true,"needEscape":true,"source":"Initializes a new instance of the <xref href=\"System.Security.Cryptography.CryptographicUnexpectedOperationException\"></xref> class with serialized data."},{"pos":[7254,7367],"content":"This constructor is called during deserialization to reconstitute the exception object transmitted over a stream.","needQuote":true,"needEscape":true,"extradata":"MT","nodes":[{"content":"This constructor is called during deserialization to reconstitute the exception object transmitted over a stream.","pos":[0,113]}]},{"pos":[7662,7711],"content":"The object that holds the serialized object data.","needQuote":true,"needEscape":true,"nodes":[{"content":"The object that holds the serialized object data.","pos":[0,49]}]},{"pos":[7807,7866],"content":"The contextual information about the source or destination.","needQuote":true,"needEscape":true,"nodes":[{"content":"The contextual information about the source or destination.","pos":[0,59]}]},{"pos":[8689,8927],"content":"Initializes a new instance of the <bpt id=\"p1\">&lt;xref href=\"System.Security.Cryptography.CryptographicUnexpectedOperationException\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> class with a specified error message and a reference to the inner exception that is the cause of this exception.","needQuote":true,"needEscape":true,"source":"Initializes a new instance of the <xref href=\"System.Security.Cryptography.CryptographicUnexpectedOperationException\"></xref> class with a specified error message and a reference to the inner exception that is the cause of this exception."},{"pos":[8940,9727],"content":"An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property. The <xref:System.Exception.InnerException%2A> property returns the same value that is passed into the constructor, or `null` if the <xref:System.Exception.InnerException%2A> property does not supply the inner exception value to the constructor.       The following table shows the initial property values for an instance of <xref:System.Security.Cryptography.CryptographicUnexpectedOperationException>.      |Property|Value|   |--------------|-----------|   |<xref:System.Exception.InnerException%2A>|The inner exception reference.|   |<xref:System.Exception.Message%2A>|The error message string.|","needQuote":false,"needEscape":true,"extradata":"MT","nodes":[{"content":"An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the &lt;xref:System.Exception.InnerException%2A&gt; property.","pos":[0,182],"source":"An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property."},{"content":"The &lt;xref:System.Exception.InnerException%2A&gt; property returns the same value that is passed into the constructor, or <ph id=\"ph1\">`null`</ph> if the &lt;xref:System.Exception.InnerException%2A&gt; property does not supply the inner exception value to the constructor.","pos":[183,427],"source":" The <xref:System.Exception.InnerException%2A> property returns the same value that is passed into the constructor, or `null` if the <xref:System.Exception.InnerException%2A> property does not supply the inner exception value to the constructor."},{"content":"The following table shows the initial property values for an instance of &lt;xref:System.Security.Cryptography.CryptographicUnexpectedOperationException&gt;.","pos":[434,585],"source":"       The following table shows the initial property values for an instance of <xref:System.Security.Cryptography.CryptographicUnexpectedOperationException>."},{"content":"|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|The inner exception reference.|   |&lt;xref:System.Exception.Message%2A&gt;|The error message string.|","pos":[591,780],"source":"      |Property|Value|   |--------------|-----------|   |<xref:System.Exception.InnerException%2A>|The inner exception reference.|   |<xref:System.Exception.Message%2A>|The error message string.|"}]},{"pos":[10567,10628],"content":"The error message that explains the reason for the exception.","needQuote":true,"needEscape":true,"nodes":[{"content":"The error message that explains the reason for the exception.","pos":[0,61]}]},{"pos":[10693,10984],"content":"The exception that is the cause of the current exception. If the `inner` parameter is not <xref uid=\"langword_csharp_null\" name=\"null\" href=\"\"></xref>, the current exception is raised in a <xref uid=\"langword_csharp_catch\" name=\"catch\" href=\"\"></xref> block that handles the inner exception.","needQuote":true,"needEscape":true,"nodes":[{"content":"The exception that is the cause of the current exception.","pos":[0,57]},{"content":"If the <ph id=\"ph1\">`inner`</ph> parameter is not <bpt id=\"p1\">&lt;xref uid=\"langword_csharp_null\" name=\"null\" href=\"\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept>, the current exception is raised in a <bpt id=\"p2\">&lt;xref uid=\"langword_csharp_catch\" name=\"catch\" href=\"\"&gt;</bpt><ept id=\"p2\">&lt;/xref&gt;</ept> block that handles the inner exception.","pos":[58,291],"source":" If the `inner` parameter is not <xref uid=\"langword_csharp_null\" name=\"null\" href=\"\"></xref>, the current exception is raised in a <xref uid=\"langword_csharp_catch\" name=\"catch\" href=\"\"></xref> block that handles the inner exception."}]},{"pos":[11792,11979],"content":"Initializes a new instance of the <bpt id=\"p1\">&lt;xref href=\"System.Security.Cryptography.CryptographicUnexpectedOperationException\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> class with a specified error message in the specified format.","needQuote":true,"needEscape":true,"source":"Initializes a new instance of the <xref href=\"System.Security.Cryptography.CryptographicUnexpectedOperationException\"></xref> class with a specified error message in the specified format."},{"pos":[11992,12320],"content":"The following table shows the initial property values for an instance of <xref:System.Security.Cryptography.CryptographicUnexpectedOperationException>.      |Property|Value|   |--------------|-----------|   |<xref:System.Exception.InnerException%2A>|`null`.|   |<xref:System.Exception.Message%2A>|The error message string.|","needQuote":false,"needEscape":true,"extradata":"MT","nodes":[{"content":"The following table shows the initial property values for an instance of &lt;xref:System.Security.Cryptography.CryptographicUnexpectedOperationException&gt;.","pos":[0,151],"source":"The following table shows the initial property values for an instance of <xref:System.Security.Cryptography.CryptographicUnexpectedOperationException>."},{"content":"|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|<ph id=\"ph1\">`null`</ph>.|   |&lt;xref:System.Exception.Message%2A&gt;|The error message string.|","pos":[157,323],"source":"      |Property|Value|   |--------------|-----------|   |<xref:System.Exception.InnerException%2A>|`null`.|   |<xref:System.Exception.Message%2A>|The error message string.|"}]},{"pos":[13145,13189],"content":"The format used to output the error message.","needQuote":true,"needEscape":true,"nodes":[{"content":"The format used to output the error message.","pos":[0,44]}]},{"pos":[13252,13313],"content":"The error message that explains the reason for the exception.","needQuote":true,"needEscape":true,"nodes":[{"content":"The error message that explains the reason for the exception.","pos":[0,61]}]}],"content":"### YamlMime:ManagedReference\nitems:\n- uid: System.Security.Cryptography.CryptographicUnexpectedOperationException\n  id: CryptographicUnexpectedOperationException\n  children:\n  - System.Security.Cryptography.CryptographicUnexpectedOperationException.#ctor\n  - System.Security.Cryptography.CryptographicUnexpectedOperationException.#ctor(System.String)\n  - System.Security.Cryptography.CryptographicUnexpectedOperationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  - System.Security.Cryptography.CryptographicUnexpectedOperationException.#ctor(System.String,System.Exception)\n  - System.Security.Cryptography.CryptographicUnexpectedOperationException.#ctor(System.String,System.String)\n  langs:\n  - csharp\n  name: CryptographicUnexpectedOperationException\n  nameWithType: CryptographicUnexpectedOperationException\n  fullName: System.Security.Cryptography.CryptographicUnexpectedOperationException\n  type: Class\n  summary: The exception that is thrown when an unexpected operation occurs during a cryptographic operation.\n  remarks: \"CryptographicUnexpectedOperationException uses the HRESULT CORSEC_E_CRYPTO_UNEX_OPER, which has the value 0x80131431.  \\n  \\n For a list of initial property values for an instance of CryptographicUnexpectedOperationException, see the constructor.\"\n  example:\n  - \"The following code example demonstrates how to use members of the CryptographicUnexpectedOperationException class.  \\n  \\n [!code-vb[System.Security.Cryptography.CryptographicUnexpectedOperationException#2](~/add/codesnippet/visualbasic/t-system.security.crypto_109_1.vb)]\\n [!code-cs[System.Security.Cryptography.CryptographicUnexpectedOperationException#2](~/add/codesnippet/csharp/t-system.security.crypto_109_1.cs)]\"\n  syntax:\n    content: >-\n      [System.Runtime.InteropServices.ComVisible(true)]\n\n      public class CryptographicUnexpectedOperationException : System.Security.Cryptography.CryptographicException\n  inheritance:\n  - System.Exception\n  - System.SystemException\n  - System.Security.Cryptography.CryptographicException\n  implements: []\n  inheritedMembers: []\n  platform:\n  - net462\n- uid: System.Security.Cryptography.CryptographicUnexpectedOperationException.#ctor\n  id: '#ctor'\n  parent: System.Security.Cryptography.CryptographicUnexpectedOperationException\n  langs:\n  - csharp\n  name: CryptographicUnexpectedOperationException()\n  nameWithType: CryptographicUnexpectedOperationException.CryptographicUnexpectedOperationException()\n  fullName: System.Security.Cryptography.CryptographicUnexpectedOperationException.CryptographicUnexpectedOperationException()\n  type: Constructor\n  assemblies:\n  - mscorlib\n  namespace: System.Security.Cryptography\n  summary: Initializes a new instance of the <xref href=\"System.Security.Cryptography.CryptographicUnexpectedOperationException\"></xref> class with default properties.\n  remarks: \"The following table shows the initial property values for an instance of <xref:System.Security.Cryptography.CryptographicUnexpectedOperationException>.  \\n  \\n|Property|Value|  \\n|--------------|-----------|  \\n|<xref:System.Exception.InnerException%2A>|`null`.|  \\n|<xref:System.Exception.Message%2A>|The localized error message string.|\"\n  example:\n  - \"The following code example demonstrates how to construct a <xref:System.Security.Cryptography.CryptographicUnexpectedOperationException> with no parameters. This code example is part of a larger example provided for the <xref:System.Security.Cryptography.CryptographicUnexpectedOperationException> class.  \\n  \\n [!code-vb[System.Security.Cryptography.CryptographicUnexpectedOperationException#1](~/add/codesnippet/visualbasic/d7702a92-c21c-4d5a-808b-_1.vb)]\\n [!code-cs[System.Security.Cryptography.CryptographicUnexpectedOperationException#1](~/add/codesnippet/csharp/d7702a92-c21c-4d5a-808b-_1.cs)]\"\n  syntax:\n    content: public CryptographicUnexpectedOperationException ();\n    parameters: []\n  overload: System.Security.Cryptography.CryptographicUnexpectedOperationException.#ctor*\n  exceptions: []\n  platform:\n  - net462\n- uid: System.Security.Cryptography.CryptographicUnexpectedOperationException.#ctor(System.String)\n  id: '#ctor(System.String)'\n  parent: System.Security.Cryptography.CryptographicUnexpectedOperationException\n  langs:\n  - csharp\n  name: CryptographicUnexpectedOperationException(String)\n  nameWithType: CryptographicUnexpectedOperationException.CryptographicUnexpectedOperationException(String)\n  fullName: System.Security.Cryptography.CryptographicUnexpectedOperationException.CryptographicUnexpectedOperationException(String)\n  type: Constructor\n  assemblies:\n  - mscorlib\n  namespace: System.Security.Cryptography\n  summary: Initializes a new instance of the <xref href=\"System.Security.Cryptography.CryptographicUnexpectedOperationException\"></xref> class with a specified error message.\n  remarks: \"The following table shows the initial property values for an instance of <xref:System.Security.Cryptography.CryptographicUnexpectedOperationException>.  \\n  \\n|Property|Value|  \\n|--------------|-----------|  \\n|<xref:System.Exception.InnerException%2A>|`null`.|  \\n|<xref:System.Exception.Message%2A>|The error message string.|\"\n  example:\n  - \"The following code example demonstrates how to construct a <xref:System.Security.Cryptography.CryptographicUnexpectedOperationException> using a custom error message. This code example is part of a larger example provided for the <xref:System.Security.Cryptography.CryptographicUnexpectedOperationException> class.  \\n  \\n [!code-vb[System.Security.Cryptography.CryptographicUnexpectedOperationException#3](~/add/codesnippet/visualbasic/ff59b282-5973-4536-ab28-_1.vb)]\\n [!code-cs[System.Security.Cryptography.CryptographicUnexpectedOperationException#3](~/add/codesnippet/csharp/ff59b282-5973-4536-ab28-_1.cs)]\"\n  syntax:\n    content: public CryptographicUnexpectedOperationException (string message);\n    parameters:\n    - id: message\n      type: System.String\n      description: The error message that explains the reason for the exception.\n  overload: System.Security.Cryptography.CryptographicUnexpectedOperationException.#ctor*\n  exceptions: []\n  platform:\n  - net462\n- uid: System.Security.Cryptography.CryptographicUnexpectedOperationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'\n  parent: System.Security.Cryptography.CryptographicUnexpectedOperationException\n  langs:\n  - csharp\n  name: CryptographicUnexpectedOperationException(SerializationInfo,StreamingContext)\n  nameWithType: CryptographicUnexpectedOperationException.CryptographicUnexpectedOperationException(SerializationInfo,StreamingContext)\n  fullName: System.Security.Cryptography.CryptographicUnexpectedOperationException.CryptographicUnexpectedOperationException(SerializationInfo,StreamingContext)\n  type: Constructor\n  assemblies:\n  - mscorlib\n  namespace: System.Security.Cryptography\n  summary: Initializes a new instance of the <xref href=\"System.Security.Cryptography.CryptographicUnexpectedOperationException\"></xref> class with serialized data.\n  remarks: This constructor is called during deserialization to reconstitute the exception object transmitted over a stream.\n  syntax:\n    content: protected CryptographicUnexpectedOperationException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);\n    parameters:\n    - id: info\n      type: System.Runtime.Serialization.SerializationInfo\n      description: The object that holds the serialized object data.\n    - id: context\n      type: System.Runtime.Serialization.StreamingContext\n      description: The contextual information about the source or destination.\n  overload: System.Security.Cryptography.CryptographicUnexpectedOperationException.#ctor*\n  exceptions: []\n  platform:\n  - net462\n- uid: System.Security.Cryptography.CryptographicUnexpectedOperationException.#ctor(System.String,System.Exception)\n  id: '#ctor(System.String,System.Exception)'\n  parent: System.Security.Cryptography.CryptographicUnexpectedOperationException\n  langs:\n  - csharp\n  name: CryptographicUnexpectedOperationException(String,Exception)\n  nameWithType: CryptographicUnexpectedOperationException.CryptographicUnexpectedOperationException(String,Exception)\n  fullName: System.Security.Cryptography.CryptographicUnexpectedOperationException.CryptographicUnexpectedOperationException(String,Exception)\n  type: Constructor\n  assemblies:\n  - mscorlib\n  namespace: System.Security.Cryptography\n  summary: Initializes a new instance of the <xref href=\"System.Security.Cryptography.CryptographicUnexpectedOperationException\"></xref> class with a specified error message and a reference to the inner exception that is the cause of this exception.\n  remarks: \"An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property. The <xref:System.Exception.InnerException%2A> property returns the same value that is passed into the constructor, or `null` if the <xref:System.Exception.InnerException%2A> property does not supply the inner exception value to the constructor.  \\n  \\n The following table shows the initial property values for an instance of <xref:System.Security.Cryptography.CryptographicUnexpectedOperationException>.  \\n  \\n|Property|Value|  \\n|--------------|-----------|  \\n|<xref:System.Exception.InnerException%2A>|The inner exception reference.|  \\n|<xref:System.Exception.Message%2A>|The error message string.|\"\n  example:\n  - \"The following code example demonstrates how to construct a <xref:System.Security.Cryptography.CryptographicUnexpectedOperationException> using a custom error message and an inner exception. This code example is part of a larger example provided for the <xref:System.Security.Cryptography.CryptographicUnexpectedOperationException> class.  \\n  \\n [!code-vb[System.Security.Cryptography.CryptographicUnexpectedOperationException#4](~/add/codesnippet/visualbasic/e5a0a5ec-6f91-4a0f-a761-_1.vb)]\\n [!code-cs[System.Security.Cryptography.CryptographicUnexpectedOperationException#4](~/add/codesnippet/csharp/e5a0a5ec-6f91-4a0f-a761-_1.cs)]\"\n  syntax:\n    content: public CryptographicUnexpectedOperationException (string message, Exception inner);\n    parameters:\n    - id: message\n      type: System.String\n      description: The error message that explains the reason for the exception.\n    - id: inner\n      type: System.Exception\n      description: The exception that is the cause of the current exception. If the `inner` parameter is not <xref uid=\"langword_csharp_null\" name=\"null\" href=\"\"></xref>, the current exception is raised in a <xref uid=\"langword_csharp_catch\" name=\"catch\" href=\"\"></xref> block that handles the inner exception.\n  overload: System.Security.Cryptography.CryptographicUnexpectedOperationException.#ctor*\n  exceptions: []\n  platform:\n  - net462\n- uid: System.Security.Cryptography.CryptographicUnexpectedOperationException.#ctor(System.String,System.String)\n  id: '#ctor(System.String,System.String)'\n  parent: System.Security.Cryptography.CryptographicUnexpectedOperationException\n  langs:\n  - csharp\n  name: CryptographicUnexpectedOperationException(String,String)\n  nameWithType: CryptographicUnexpectedOperationException.CryptographicUnexpectedOperationException(String,String)\n  fullName: System.Security.Cryptography.CryptographicUnexpectedOperationException.CryptographicUnexpectedOperationException(String,String)\n  type: Constructor\n  assemblies:\n  - mscorlib\n  namespace: System.Security.Cryptography\n  summary: Initializes a new instance of the <xref href=\"System.Security.Cryptography.CryptographicUnexpectedOperationException\"></xref> class with a specified error message in the specified format.\n  remarks: \"The following table shows the initial property values for an instance of <xref:System.Security.Cryptography.CryptographicUnexpectedOperationException>.  \\n  \\n|Property|Value|  \\n|--------------|-----------|  \\n|<xref:System.Exception.InnerException%2A>|`null`.|  \\n|<xref:System.Exception.Message%2A>|The error message string.|\"\n  example:\n  - \"The following code example demonstrates how to construct a <xref:System.Security.Cryptography.CryptographicUnexpectedOperationException> using a time format and the current date. This code example is part of a larger example provided for the <xref:System.Security.Cryptography.CryptographicUnexpectedOperationException> class.  \\n  \\n [!code-vb[System.Security.Cryptography.CryptographicUnexpectedOperationException#5](~/add/codesnippet/visualbasic/54b79c0c-e5ea-4123-a712-_1.vb)]\\n [!code-cs[System.Security.Cryptography.CryptographicUnexpectedOperationException#5](~/add/codesnippet/csharp/54b79c0c-e5ea-4123-a712-_1.cs)]\"\n  syntax:\n    content: public CryptographicUnexpectedOperationException (string format, string insert);\n    parameters:\n    - id: format\n      type: System.String\n      description: The format used to output the error message.\n    - id: insert\n      type: System.String\n      description: The error message that explains the reason for the exception.\n  overload: System.Security.Cryptography.CryptographicUnexpectedOperationException.#ctor*\n  exceptions: []\n  platform:\n  - net462\nreferences:\n- uid: System.Security.Cryptography.CryptographicException\n  isExternal: false\n  name: System.Security.Cryptography.CryptographicException\n- uid: System.Security.Cryptography.CryptographicUnexpectedOperationException.#ctor\n  parent: System.Security.Cryptography.CryptographicUnexpectedOperationException\n  isExternal: false\n  name: CryptographicUnexpectedOperationException()\n  nameWithType: CryptographicUnexpectedOperationException.CryptographicUnexpectedOperationException()\n  fullName: System.Security.Cryptography.CryptographicUnexpectedOperationException.CryptographicUnexpectedOperationException()\n- uid: System.Security.Cryptography.CryptographicUnexpectedOperationException.#ctor(System.String)\n  parent: System.Security.Cryptography.CryptographicUnexpectedOperationException\n  isExternal: false\n  name: CryptographicUnexpectedOperationException(String)\n  nameWithType: CryptographicUnexpectedOperationException.CryptographicUnexpectedOperationException(String)\n  fullName: System.Security.Cryptography.CryptographicUnexpectedOperationException.CryptographicUnexpectedOperationException(String)\n- uid: System.String\n  parent: System\n  isExternal: true\n  name: String\n  nameWithType: String\n  fullName: System.String\n- uid: System.Security.Cryptography.CryptographicUnexpectedOperationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  parent: System.Security.Cryptography.CryptographicUnexpectedOperationException\n  isExternal: false\n  name: CryptographicUnexpectedOperationException(SerializationInfo,StreamingContext)\n  nameWithType: CryptographicUnexpectedOperationException.CryptographicUnexpectedOperationException(SerializationInfo,StreamingContext)\n  fullName: System.Security.Cryptography.CryptographicUnexpectedOperationException.CryptographicUnexpectedOperationException(SerializationInfo,StreamingContext)\n- uid: System.Runtime.Serialization.SerializationInfo\n  parent: System.Runtime.Serialization\n  isExternal: false\n  name: SerializationInfo\n  nameWithType: SerializationInfo\n  fullName: System.Runtime.Serialization.SerializationInfo\n- uid: System.Runtime.Serialization.StreamingContext\n  parent: System.Runtime.Serialization\n  isExternal: true\n  name: StreamingContext\n  nameWithType: StreamingContext\n  fullName: System.Runtime.Serialization.StreamingContext\n- uid: System.Security.Cryptography.CryptographicUnexpectedOperationException.#ctor(System.String,System.Exception)\n  parent: System.Security.Cryptography.CryptographicUnexpectedOperationException\n  isExternal: false\n  name: CryptographicUnexpectedOperationException(String,Exception)\n  nameWithType: CryptographicUnexpectedOperationException.CryptographicUnexpectedOperationException(String,Exception)\n  fullName: System.Security.Cryptography.CryptographicUnexpectedOperationException.CryptographicUnexpectedOperationException(String,Exception)\n- uid: System.Exception\n  parent: System\n  isExternal: true\n  name: Exception\n  nameWithType: Exception\n  fullName: System.Exception\n- uid: System.Security.Cryptography.CryptographicUnexpectedOperationException.#ctor(System.String,System.String)\n  parent: System.Security.Cryptography.CryptographicUnexpectedOperationException\n  isExternal: false\n  name: CryptographicUnexpectedOperationException(String,String)\n  nameWithType: CryptographicUnexpectedOperationException.CryptographicUnexpectedOperationException(String,String)\n  fullName: System.Security.Cryptography.CryptographicUnexpectedOperationException.CryptographicUnexpectedOperationException(String,String)\n- uid: System.Security.Cryptography.CryptographicUnexpectedOperationException.#ctor*\n  parent: System.Security.Cryptography.CryptographicUnexpectedOperationException\n  isExternal: false\n  name: CryptographicUnexpectedOperationException\n  nameWithType: CryptographicUnexpectedOperationException.CryptographicUnexpectedOperationException\n"}